readme.md

```diff
@@ -48,14 +48,27 @@ Default configuration
       "exec": false
     },
     "bash": "%action% %args%",
-    "composer": "%action% %args%",
+    "composer": "composer %action% %args%",
     "node": {
-      "command": "%action% %args%",
+      "command": "node %action% %args%",
       "user": "node"
     },
-    "npm": "%action% %args%",
-    "git": "%action% %args%",
-    "yarn": "%action% %args%"
+    "npm": "npm %action% %args%",
+    "git": "git %action% %args%",
+    "yarn": "yarn %action% %args%",
+    "multiple-cmd": {
+      "command": ["echo multiple command as array", "@nested-cmd arg1 arg2"]
+    },
+    "nested-cmd": {
+      "command": ["echo nested command %args%", "@deepnested-cmd --opt1 val1 --opt2 val2"]
+    },
+    "deepnested-cmd": {
+      "command": ["echo deep nested command %args%"]
+    },
+    "host-cmd": {
+      "service": "@host",
+      "command": "docker-compose version"
+    }
```

Comment:
good examples, wrong position :) you're editing the display of the `default configuration` ^^